docs/docs/guides/token-expiration.mdx

```diff
@@ -75,6 +75,44 @@ for refresh tokens to never expire.
     #....
 ```
 
+### Refresh Token Rotation
+
+When a refresh token is used it is deactivated, which is known as Refresh Token
+Rotation. By default Hydra deactivates the refresh token it receives and issues
```

Comment:
Maybe not that reusing the token will also cause that all tokens related to that refresh token are invalidated!